documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -0,0 +1,112 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrading-kafka-versions.adoc
+
+[id='proc-upgrading-brokers-newer-kafka-{context}']
+
+= Upgrading brokers to a newer Kafka version
+
+This procedure describes how to upgrade a {ProductName} Kafka cluster from one version to a higher version; for example 2.0.0 to 2.1.0.
+
+.Prerequisites
+
+* The Cluster Operator, which supports both versions of Kafka, is up and running.
+* A `Kafka` resource to be upgraded.
+* You have checked that your `Kafka.spec.kafka.config` contains no options that are not supported in the version of Kafka that you are upgrading to.
+
+.Procedure
+
+. Consult the table below and determine whether the new Kafka version has a different log message format version than the old version. 
++
+include::snip-kafka-versions.adoc[leveloffset=+1]
++
+If the log message format versions are the same proceed to the next step. 
+Otherwise ensure the `Kafka.spec.kafka.config` has the `log.message.format.version` configured to the default for the old version.
```

Comment:
```suggestion
Otherwise, ensure the `Kafka.spec.kafka.config` has the `log.message.format.version` configured to the default for the previous version.
```